lib/matplotlib/ticker.py

```diff
@@ -168,6 +169,15 @@
     long = int
 
 
+# Work around numpy/numpy#6127.
+def divmod(x, y):
```

Comment:
It's not listed in `__all__`, just like a bunch of other functions in the same module (`closeto`, `decade_{up,down}`, `nearest_long`, etc.) which are not prefixed by an underscore but still arguably private.